src/uucore/coreopts.rs

```diff
@@ -139,3 +139,35 @@ macro_rules! new_coreopts {
         })
     );
 }
+
+#[inline]
+pub fn is_stdin(files: &Vec<String>) -> bool {
+    files.is_empty() || &files[0] == "-"
```

Comment:
As I mentioned in this PR and the other PR, `files[0] == "-"` doesn't cover everything.